.travis.yml

```diff
@@ -17,9 +17,6 @@ script:
 - GOOS=darwin GOARCH=amd64 go build -ldflags="-X 'github.com/sonatype-nexus-community/ahab/buildversion.BuildVersion=$VERSION'
   -X 'github.com/sonatype-nexus-community/ahab/buildversion.BuildTime=$time' -X 'github.com/sonatype-nexus-community/ahab/buildversion.BuildCommit=$TRAVIS_COMMIT'"
   -o ahab-darwin.amd64-$VERSION
-- GOOS=windows GOARCH=amd64 go build -ldflags="-X 'github.com/sonatype-nexus-community/ahab/buildversion.BuildVersion=$VERSION'
```

Comment:
So i removed this. Mostly b/c i used a SysCall that doesn't work on Windows but does work on mac and linux. Unless im missing something i dont see how ahab would be useful on windows. So thus my decision to remove it. Please let me know if this assumption is incorrect.

Reply:
That's great. I think it was there because I copied it from the `nancy` setup!